src/EventListener/Pimcore/ChangeListener.php

```diff
@@ -66,13 +71,13 @@ public static function getSubscribedEvents(): array
         return [
             AssetEvents::POST_ADD => 'handle',
             AssetEvents::POST_UPDATE => 'handle',
-            AssetEvents::PRE_DELETE => 'handle',
+            AssetEvents::POST_DELETE => 'handle',
```

Comment:
Why is this needed?

Reply:
after our talk i had another tough on this. Usually it should not matter if it's PRE or POST Delete. We simply have to make sure, the Queue is not processed before the Item is deleted. This could happen if the queue is empty (likely what happened during my tests when nothing happened).
So either we stick to POST_DELETE, or we handle the deletion Event in an `DeleteElement` Message which we already agreed on would not be advisable. So I propose to change this to POST_DELETE as it would not change the behaviour as it usually is processed after the Element is deleted.